core/shared/src/main/scala/zio/Chunk.scala

```diff
@@ -821,7 +898,14 @@ sealed abstract class Chunk[+A] extends ChunkLike[A] { self =>
   /**
    * Takes all elements so long as the effectual predicate returns true.
    */
+  @deprecated("use takeWhileZIO", "2.0.0")
```

Comment:
Thank you for all the `@deprecated` warnings! It will help early users of 2.0 release candidates.